packages/components/pagination/src/pagination.ts

```diff
@@ -186,6 +189,9 @@ export default defineComponent({
     const { t } = useLocale()
     const ns = useNamespace('pagination')
     const vnodeProps = getCurrentInstance()!.vnode.props || {}
+    const _size = useFormSize(
+      computed(() => (props.small ? 'small' : vnodeProps?.size))
+    )
```

Comment:
The Pagination component should not belong to the form component, and I don't quite understand why it is handled this way

Reply:
I think 'useFormSize' can be managed uniformly, and this will also work if the user puts it in 'from'

Reply:
However, it seems that the situation where `pagination` is placed inside `form` will not occur. 🤔